queries/r/textobjects.scm

```diff
@@ -79,44 +67,44 @@
   .
   (_) @parameter.inner
   .
-  ","? @_end)
+  (comma)? @_end)
   (#make-range! "parameter.outer" @parameter.inner @_end))
 
 ; number
 (float) @number.inner
 
 ; assignment
-(left_assignment
+("<-"
   name: (_) @assignment.lhs
   value: (_) @assignment.inner @assignment.rhs) @assignment.outer
 
-(left_assignment
+("<-"
   name: (_) @assignment.inner)
 
-(right_assignment
+("->"
   value: (_) @assignment.inner @assignment.lhs
   name: (_) @assignment.rhs) @assignment.outer
 
-(right_assignment
+("->"
   name: (_) @assignment.inner)
 
-(equals_assignment
+((comma)
   name: (_) @assignment.lhs
   value: (_) @assignment.inner @assignment.rhs) @assignment.outer
 
-(equals_assignment
+((comma)
   name: (_) @assignment.inner)
 
-(super_assignment
+("<<-"
   name: (_) @assignment.lhs
   value: (_) @assignment.inner @assignment.rhs) @assignment.outer
 
-(super_assignment
+("<<-"
   name: (_) @assignment.inner)
 
-(super_right_assignment
+("->>"
   value: (_) @assignment.inner @assignment.lhs
   name: (_) @assignment.rhs) @assignment.outer
 
-(super_right_assignment
-  name: (_) @assignment.inner)
+(binary_operator)
```

Comment:
Failure is indeed intended because there is a superfluous `)` here.

Reply:
This is in the current version, right? `super_right_assignment` does not exist anymore.

Reply:
Yes, the format of the _current_ version is an incorrect s-expression. The way you've written it, the `name` is _outside_ the pattern for the `(binary_operator)` node, which makes no sense. (Compare with the paren placement in the old version.)